configure

```diff
@@ -191,6 +191,9 @@ my @specs = # Array of array-refs, each inner array is an option group which
     'inline!', 'Inlining (yes)',
     'static!', 'Static libraries (no)',
     'ipv6!', 'IPv6 support (no)',
+    'asan!', 'Address Sanitizer (no)',
+    'tsan!', 'Thread Sanitizer (no)',
+    'ubsan!', 'Undefined Behavior Sanitizer (no)',
```

Comment:
Should we make these a single option `--sanitize=foo`?

Reply:
If the sanitizers were all mutably exclusive I would say yes, but ubsan can be safely combined with asan as [encouraged when using libfuzzer](https://llvm.org/docs/LibFuzzer.html#fuzzer-usage)

Reply:
The options processor supports multi-value.  I don't remember if it's `--sanitize=a --sanitize=b` or `--sanitize=a,b` or either of those.

Reply:
Yes, I can do that. I think it's the first one that's builtin, but we could make work like the second one if we really wanted.